docs/index.rst

```diff
@@ -29,14 +29,6 @@ Features
 * De-duplication of all saved content
 * Support disconnected and air-gapped environments with the Pulp Import/Export facility for container repositories
 
-Tech Preview
```

Comment:
If there will be any features in tech preview, I will add the paragraph back. Right now, it only clutters the documentation because we do not have any features in tech preview.

Reply:
see https://github.com/pulp/pulp_container/pull/1287#discussion_r1202671587
I don't consider cluttering docs with one 3 liner paragraph.

Reply:
What would even be better would be a line like "There are no features currently in tech preview" I think.